core/src/test/java/kafka/test/MockController.java

```diff
@@ -77,8 +82,28 @@ private MockController(Collection<MockTopic> initialTopics) {
     }
 
     @Override
-    public CompletableFuture<CreateTopicsResponseData> createTopics(CreateTopicsRequestData request) {
-        throw new UnsupportedOperationException();
+    synchronized public CompletableFuture<CreateTopicsResponseData>
+            createTopics(CreateTopicsRequestData request) {
+        CreateTopicsResponseData response = new CreateTopicsResponseData();
+        for (CreatableTopic topic : request.topics()) {
+            if (topics.containsKey(topic.name())) {
```

Comment:
the key type of `topics` is `Uuid` and hence this check is weird. Maybe it should be replaced by `topicNameToId`

Reply:
Good catch.  I'll fix this